ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueCommitting.scala

```diff
@@ -207,4 +216,37 @@ object KeyValueCommitting {
 
     }
   }
+
+  private object Metrics {
```

Comment:
The `var`s in here should probably be marked as `@volatile`.

Reply:
In fact, it's probably best to create a custom `Gauge` implementation that always reports the last value that the application sets, and register that gauge directly with `MetricRegistry#register`.

Reply:
Thanks, Added `VarGauge`